src/components/TextField.re

```diff
@@ -0,0 +1,48 @@
+let component = ReasonReact.statelessComponent("Input");
+
+let make = (~placeholder="", ~value, ~onChange, _children) => {
+  ...component,
+  render: _self =>
+    <div
+      className=Css.(
+        style([
+          marginBottom(px(Theme.spacingPx.xsmall)),
+          media(
+            Breakpoints.up(Md),
+            [marginBottom(px(Theme.spacingPx.small))],
+          ),
+        ])
+      )>
+      <input
+        type_="text"
+        placeholder
+        value
+        onChange
+        className=Css.(
+          style([
```

Comment:
Would be nice to throw in `maxLength` here too - probably as a prop with default value?

Reply:
Done :)